nursereg/tasks.py

```diff
@@ -317,7 +334,27 @@ def update_create_vumi_contact(nursereg_id, client=None, sender=None):
             except:
                 logger.error('Problem contacting http_api', exc_info=True)
 
-            return updated_contact
+            cm_active_nr_subs = Subscription.objects.filter(
+                to_addr=nursereg.cmsisdn, active=True,
+                message_set__short_name="nurseconnect")
```

Comment:
This is only going to work as long as there's only one nurse related messageset. I don't think we need to address this now, but please note so in a comment